src/components/Video/Video.tsx

```diff
@@ -163,13 +168,34 @@ const Video = ({
   const aspectRatioStyle = { aspectRatio: `${video.width} / ${video.height}` };
 
   const videoClasses = classNames(styles.video, {
-    [styles.newControls]: hasNewControls,
-    [styles.oldControls]: !hasNewControls,
+    [styles.newControls]: hasFocusMode,
+    [styles.oldControls]: !hasFocusMode,
   });
 
+  const getCurrentTimeParam = () => {
+    if (!player) {
+      return null;
+    }
+
+    return `?time=${player.currentTime()}`;
+  };
+
+  const handleOpenFocusMode = () => {
```

Comment:
That's true, but also keep in mind that we need the current time from the video, to pass it as a query prop, so we're able to continue when we open the page. So, it can be a handler that we call from inside the Video component, but as i see it, we still need to handle the click on the Video component first.